content/shmem_pcontrol.tex

```diff
@@ -0,0 +1,54 @@
+\apisummary{
+  Allows the user to control profiling.
+}
+
+\begin{apidefinition}
+
+\begin{Csynopsis}
+void @\FuncDecl{shmem\_pcontrol}@(const int level, ...);
```

Comment:
The use of const qualifier for a value parameter may be useful per [this](https://stackoverflow.com/questions/8714250/isnt-const-redundant-when-passing-by-value) post on SO.

Reply:
Some [MPI rationale](https://lists.mpi-forum.org/pipermail/mpi-forum/2017-April/006567.html) on the use of `const int` vs. `int`.

Reply:
Quoting Bill Gropp 

> I think that in looking at the language-independent version, the IN was noted, and const was added without thinking it through.

Without thinking is through(!) - and we just piggyback what MPI does? Let just fix this in the chapter committee changes.

Reply:
An argument for using the same function signature as `MPI_Pcontrol` is that it may make it easier to create tools that support both MPI and OpenSHMEM.